### DISCUSSION TITLE
dark theme

### DISCUSSION THREAD

**Discussion by shivam-devs** [refinedev/refine#3783] [Q&A]
why this refine app  is always showing dark theme even if my browser is not in dark theme?
![Annotation 2023-03-10 181031](https://user-images.githubusercontent.com/93398933/224320218-2af85a4a-27b9-46d7-9e73-c946f2b2eeaa.png)
help me.

**Comment by vinclou** [comment]
@Biswa098 The question is too wide and ambiguous, I would bet there's something wrong you have configured in `App.tsx` specifically the theme provider and answer depends on what component framework you've picked and how this configuration looks like

**Comment by BatuhanW** [comment]
Thank you for contacting us! Can you provide a CodeSandbox or StackBlitz environment that we can use to reproduce the issue?
